include/macis/gf/inn_prods.hpp

```diff
@@ -32,14 +32,7 @@ namespace macis {
  */
 inline double MyInnProd(const std::vector<double>& vecR,
```

Comment:
At some point it might make sense to just remove these functions entirely since all overloads call the same `blaspp` function. Not a high-priority now, but something to keep in mind.

Reply:
Would you rather have the full, lengthy `blaspp` call everytime, or a single wrapper to make it a little easier to read?

Reply:
Globally, yes, but having local (e.g. lambda) aliases would be fine.